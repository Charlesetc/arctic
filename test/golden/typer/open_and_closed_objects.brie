[
  define bob <name = 2 , style = 3>
  bob.name
  bob.style

  define bob <name = "Hi">
]
