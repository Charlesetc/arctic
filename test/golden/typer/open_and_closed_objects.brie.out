root[(ident(define):: ident(main):: block[][(block[][(ident(define):: ident(bob):: <name = (ident(2)::):: , style = (ident(3)::)::>::)::, (ident(bob)::.name::)::, (ident(bob)::.style::)::, (ident(define):: ident(bob):: <name = (string(Hi)::)::>::)::]::Function)::Function]::Function)::]::